InventoryManagementSystem/Inventory.cs

```diff
@@ -0,0 +1,103 @@
+﻿
+
+using System;
+namespace InventoryManagementSystem
+{
+	public class Inventory
+	{
+		private List<Product> products = new List<Product>();
+		public Inventory()
+		{
+		
+		}
+
+		public void AddProduct(Product product)
+		{
+			products.Add(product);
+			Console.WriteLine("Product added successfully.");
+		}
+
+		public void ViewProducts()
+		{
+			if(products.Count() == 0)
+			{
+				Console.WriteLine("No products.");
+				return;
+			}
+
+			foreach (var product in products)
+			{
+				Console.WriteLine(product.ToString());
+			}
+		}
+		public void EditProduct(string name) 
+		{
+			bool isFound = false;
+			foreach (var product in products)
+			{
+                if (product.Name == name)
```

Comment:
If you picked another field other than name to uniquely identify entities , then this would change as well.